plugins/colorbutton/plugin.js

```diff
@@ -283,8 +282,8 @@ CKEDITOR.plugins.add( 'colorbutton', {
 				output.push( '<td>' +
 					'<a class="cke_colorbox" _cke_focus=1 hidefocus=true' +
 						' title="', colorLabel, '"' +
-						' onclick="CKEDITOR.tools.callFunction(', clickFn, ',\'', colorName, '\',\'', type, '\'); return false;"' +
-						' href="javascript:void(\'', colorLabel, '\')"' +
+						' onclick="CKEDITOR.tools.callFunction(', clickFn, ',\'#', colorCode, '\',\'', type, '\'); return false;"' +
+						' href="javascript:void(\'', colorCode, '\')"' +
```

Comment:
Probably I'm missing something, but why statusbar text was changed?

Reply:
It shows color hash when using colorbutton with default colors, so I changed it for consistency.